Winget-AutoUpdate/functions/Get-NotifLocale.ps1

```diff
@@ -22,6 +22,13 @@ Function Get-NotifLocale {
     #Get locale XML file content
     [xml]$Script:NotifLocale = Get-Content $LocaleFile -Encoding UTF8 -ErrorAction SilentlyContinue
 
+    #Test if new strings exist in $LocaleFile
```

Comment:
For me, it's easier to add English by default on all xml files (and I can handle that), and we will hope that translators will help us to translate their language file 😇😇

Reply:
or if you can do it into your PR, it would be easier I guess. Just copy/paste the part added to the other files, please :)

Reply:
I'll fix it